sdk/core/azure-core-http-netty/pom.xml

```diff
@@ -107,6 +113,7 @@
     <dependency>
       <groupId>com.github.tomakehurst</groupId>
       <artifactId>wiremock-standalone</artifactId>
+      <version>2.24.1</version>
```

Comment:
I thought explicit version numbers like this should be in our parent/pom.xml

Reply:
Yep!

Reply:
Resolved.